src/map/entities/trustentity.cpp

```diff
@@ -49,11 +49,6 @@ CTrustEntity::CTrustEntity(CCharEntity* PChar)
     PAI = std::make_unique<CAIContainer>(this, std::make_unique<CPathFind>(this), std::make_unique<CTrustController>(PChar, this), std::make_unique<CTargetFind>(this));
 }
 
-const int8* CTrustEntity::GetName()
-{
-    return (const int8*)packetName.c_str();
```

Comment:
Does the packetName still get sent in the packet to the client? As in if you cast `Trust: Selh'teus` does he pop on screen as `Selh'teus` or as `Selh_teus`/`Selh teus` ?

I think we need to do what allied mobs do, and have name and packetName as separate things, because both get used but for different purposes. Possibly this already happens because trusts are still mobs:
https://github.com/project-topaz/topaz/blob/8985d2af68876367592222836db1f8e5b15c8175/src/map/entities/mobentity.h#L244

If threes nothing to worry about then just mark my comment as resolved. :)

Reply:
Looks like it's properly pulling in the display name for the client.  I just checked Selh'teus, and his punctuation is correct.